<launch>
    <arg name="namespace"   default="red"/>
    <arg name="odom"        default="/mavros/global_position/local"/>
    <arg name="traj_ref"    default="uav/trajectory"/>

    <group ns="$(arg namespace)">
        
        <!-- Visual servo node -->
        <node
            pkg="uav_ros_control"
            type="visual_servo_node"
            name="visual_servo_node"
            output="screen">

            <!-- Topic remapping -->
            
        </node>
    </group>
</launch>
